src/server/worker/driver/driver.go

```diff
@@ -41,7 +41,7 @@ import (
 // TaskNamespace returns the namespace used by the task package for this
 // pipeline.
 func TaskNamespace(pipelineInfo *pps.PipelineInfo) string {
-	return fmt.Sprintf("/pipeline-%s/v%d", pipelineInfo.Pipeline.Name, pipelineInfo.Version)
+	return fmt.Sprintf("/pipeline-%s@v%d", pipelineInfo.Pipeline.Name, pipelineInfo.Version)
```

Comment:
Can we add the same padding we have in the pps version key (8).

Reply:
I've just reused the key function, but can copy if it's preferable to keep them separate.